extensions/wikia/CreateNewWiki/maintenance/dumpStarters.php

```diff
@@ -119,18 +123,29 @@ private function generateSqlDump( $filename, $starter ) {
 	 * @throws DumpStartersException
 	 */
 	private function storeDump( $filename, $dest ) {
+		global $wgWikiaEnvironment;
+		global $wgGcsConfig;
```

Comment:
You can declare multiple variables as global on the same line ie. `global $wgWikiaEnvironment, $wgGcsConfig;`